ui/app/AppLayouts/Chat/ChatColumn/ChatMessages.qml

```diff
@@ -257,6 +259,14 @@ SplitView {
             icon: StandardIcon.Critical
         }
 
+        Timer {
+            id: modelLoadingDelayTimer
+            interval: 1000
+            onTriggered: {
+                root.countOnStartUp = messageListDelegate.count;
```

Comment:
Do we really need an interval to do this? Wouldn't the `countOnStartUp` be binded to the count if we just did 
```qml
property int countOnStartUp: messageListDelegate.count;
```
You probably know more than me, so feel free to explain to me that I'm wrong 😅

Reply:
Yes, there is always a delay between the Component.onCompleted signal is emitted and the model being loaded (the model is done loading afterwards), it's usually 300 to 800 ms, however I used 1000 just to be on the safest side, we can decrease it if you like but as it's not a critical thing here to really catch up the exact time the model needs I believe 1000 is ok. We need to do this in Component.onCompleted so that it's done once, and thus we know that every new message is actually just added, if we use simple binding that would be evaluated each time the count changes so when a new delegate is added it will be the same count (-1) as its index.

Reply:
Ah ok got it. I also misunderstood the code and thought the Timer was called every second, but it's just one time, so that's fine.